guides/07_FAQs.md

```diff
@@ -53,6 +53,12 @@ The SDK is built to target ES2015, both syntax and features. If you need your bu
 
 Note that due to limitations in transpilers, requirements of the web platform might result in transpiled ES5 code that raises an error when run, such as "Please use the 'new' operator, this DOM object constructor cannot be called as a function". Prefer using ES2015 code on supported platforms.
 
+## Known Browser Issues
+Please refer https://github.com/aws/amazon-chime-sdk-js/issues/1059/ for all the known Browser Issues.
+
+### In Safari Audio stops playing as browser is minimized or move to background, is this a known issue?
+In [Safari](https://bugs.webkit.org/show_bug.cgi?id=231105) macOS Browser, Audio stops playing in background if audio source is binded to AudioContext API. To remediate this Issue please disable the WebAudio.
```

Comment:
Can we explicitly say the link to the bug? A bit random to make Safari as a link.

Reply:
[nit]: this Issue -> this issue
[suggestion]: Could we add some details about the version, like "in Safari 14 or 15"?

Reply:
Actually it is the same pattern used for all the bugs in the FAQ page.

Reply:
It is the issue with both safari 14 and 15.